Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -0,0 +1,4175 @@
+      name: Set Unsigned Process
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: Unsigned_Process
+      value:
+        simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2820,
+          "y": 1855
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "103":
+    id: "103"
+    taskid: b6812984-343e-4a30-8382-b711218f9a6b
+    type: title
+    task:
+      id: b6812984-343e-4a30-8382-b711218f9a6b
+      version: -1
+      name: Benign
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "58"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2050,
+          "y": 3830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "107":
+    id: "107"
+    taskid: 20238e3c-8d69-4727-8061-b63e4bf163f7
+    type: regular
+    task:
+      id: 20238e3c-8d69-4727-8061-b63e4bf163f7
+      version: -1
+      name: Check Whether The Source Host Is an FTP Server
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: FTP_Server
+      stringify:
+        simple: "true"
+      value:
+        complex:
+          root: ActiveDirectory.Computers
+          filters:
+          - - operator: match
+              left:
+                value:
+                  simple: ActiveDirectory.Computers.memberOf
+                iscontext: true
+              right:
+                value:
+                  simple: (?i)CN=.*(?:FTP|File).*,
+          accessor: name
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "109":
+    id: "109"
+    taskid: a9ef2265-0666-4514-865e-23f6b9f94b9a
+    type: regular
+    task:
+      id: a9ef2265-0666-4514-865e-23f6b9f94b9a
+      version: -1
+      name: Check Whether The Source Host Is an SMTP Server
+      description: Set a value in context under the key you entered.
+      scriptName: Set
```

Comment:
Before, I had a conditional task, but Yarden suggested I reduce it to a single task since the filters can determine if the initiator host is an SMTP server.